tests/testReadDataframe.py

```diff
@@ -0,0 +1,54 @@
+import unittest
+import pandas as pd
+import sys
+sys.path.append('../')
+from src.utils.functions import read_dataframe
+
+
+test_path = '/mnt/c/Users/Dev/desktop/mlops_project/tests/wine_data/test_wine_data.csv'
+train_path = '/mnt/c/Users/Dev/desktop/mlops_project/tests/wine_data/train_wine_data.csv'
```

Comment:
Should have used system relative path for the test and train paths, ideally, this should be mock data, not actual data. The whole point about tests is mocking data that will likely be seen in production